Makefile

```diff
@@ -1,5 +1,5 @@
 
-version = f60b92dfbf132e9991e56dd91dc99c3cec470ef7
+version = fron
```

Comment:
this needs updating. it's right at the moment but needs to be the commit on master after PR merges.